pytorch/training/docker/1.6.0/py3/Dockerfile.gpu

```diff
@@ -0,0 +1,185 @@
+# The tag for the base image is: 10.1-cudnn7-devel-ubuntu16.04
+FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu16.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON_VERSION=3.6.10
+ARG OPEN_MPI_VERSION=4.0.1
+ARG CUBLAS_VERSION=10.2.1.243-1_amd64
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=0.8.1
+
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+# Force stdin, stdout and stderr to be totally unbuffered. Good for logging
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/lib"
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/opt/conda/lib"
+ENV PYTHONIOENCODING=UTF-8
+ENV LANG=C.UTF-8
+ENV LC_ALL=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+ENV TORCH_CUDA_ARCH_LIST="3.5 5.2 6.0 6.1 7.0+PTX"
+ENV TORCH_NVCC_FLAGS="-Xfatbin -compress-all"
+ENV HOROVOD_VERSION=0.19.1
+ENV DGLBACKEND=pytorch
+ENV CMAKE_PREFIX_PATH="$(dirname $(which conda))/../"
+ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
+
+ARG PT_TRAINING_URL=https://aws-pytorch-binaries.s3-us-west-2.amazonaws.com/r1.6.0_aws/20200727-215603/a40b1c195ba679596118514fcbeae1f422591426/gpu/torch-1.6.0-cp36-cp36m-manylinux1_x86_64.whl
+
+RUN apt-get update \
+ && apt-get install -y  --allow-downgrades --allow-change-held-packages --no-install-recommends \
+    build-essential \
+    ca-certificates \
+    cmake \
+    cuda-command-line-tools-10-1 \
+    cuda-cufft-10-1 \
+    cuda-curand-10-1 \
+    cuda-cusolver-10-1 \
+    cuda-cusparse-10-1 \
+    curl \
+    emacs \
+    git \
+    jq \
+    libglib2.0-0 \
+    libgl1-mesa-glx \
+    libsm6 \
+    libxext6 \
+    libxrender-dev \
+    libgomp1 \
+    libibverbs-dev \
+    libhwloc-dev \
+    libnuma1 \
+    libnuma-dev \
+    vim \
+    wget \
+    zlib1g-dev \
+ && apt-get remove -y cuda-cufft-dev-10-1 \
+    cuda-curand-dev-10-1 \
+    cuda-cusolver-dev-10-1 \
+    cuda-npp-dev-10-1 \
+    cuda-nvgraph-dev-10-1 \
+    cuda-nvjpeg-dev-10-1 \
+    cuda-nvrtc-dev-10-1
```

Comment:
Please add statement:
```
&& rm -rf /var/lib/apt/lists/*
```
to this RUN section, to reduce layer size.

Reply:
Addressed in next update.